### PR TITLE
Website Redesign: Courses page improvement

### PR DESCRIPTION
**Pull request by JasonMun7**
### Summary 
This PR address the following changes for the Courses Page

- [x] Cursor Pointer on FAQ 
- [x] Hero Section fix (Addressed in Andrew's Merged PR)
- [x] Fixed font weights and omitted leading space
- [x] Omit card shadow (Addressed with Chris's PR #742 ) 
- [x] Align date and time in timeline
- [x] Align Icons with Text
- [x] Added Optional Profile image for the "Past Experiences" Section
- [x] Applied secondary Button variant to Learn More
- [x] Decreased Spacing In between each element of the page

### Notion/Figma Link 
[Cursor](https://www.notion.so/New-Website-COURSE-FAQ-section-cursor-pointer-14a0ad723ce180adaeacedf3bab48752?pvs=4)
[Font Weights/spaces](https://www.notion.so/New-Website-COURSE-Past-Student-Experiences-and-Past-Student-Projects-headers-font-weight-14a0ad723ce180f5a655c9611770efaa?pvs=4)
[Font Weights/spaces for Trending section](https://www.notion.so/New-Website-COURSE-Trends-in-Web-Dev-section-fixes-14a0ad723ce180a589c0c097a1a784c6?pvs=4)
[Card Shadow](https://www.notion.so/New-Website-COURSE-Course-staff-section-fixes-14a0ad723ce180f9a3bbe0505bd7d67c?pvs=4)
[Timeline fixes](https://www.notion.so/New-Website-COURSE-Timeline-section-fixes-14a0ad723ce180fc8c47cd75d8d4383b?pvs=4)
[Icon and text alignment](https://www.notion.so/New-Website-COURSE-3-features-in-row-section-fixes-14a0ad723ce180208a89ffdad2e4752e?pvs=4)
[Section Spacing](https://www.notion.so/New-Website-COURSE-Spacing-between-sections-14d0ad723ce180fbb2c9d01b3eb5f27f?pvs=4)

### Test Plan 
Trending Section Before: 
Screenshot 2024-11-26 at 12 06 36 PM
Trending Section After font weight and spacing changes:
Screenshot 2024-11-28 at 9 18 37 PM

Past Student Projects Before (Same change with all font-black to font weight of 700): 
Screenshot 2024-11-26 at 12 10 32 PM
Past Student Projects After: 
Screenshot 2024-11-28 at 9 12 10 PM

Timeline Before: 
Screenshot 2024-11-26 at 12 09 20 PM
Timeline After: 
Screenshot 2024-11-28 at 9 15 40 PM

Icon Section Before: 
Screenshot 2024-11-26 at 12 06 14 PM
Icon Section After with moving Icons to the left to be aligned: 
Screenshot 2024-11-28 at 9 17 08 PM

Section Spacing before:
Screenshot 2024-11-28 at 7 50 30 PM\
Section Spacing After: 
Screenshot 2024-11-28 at 9 19 43 PM

Testimonial Before: 
Screenshot 2024-11-28 at 9 26 42 PM
Testimonial After: 
Screenshot 2024-11-28 at 9 26 02 PM

### Notes 

- Added the Secondary Button Variant to the global css 
- For the timeline section, I made it so that it added the @ if there was a given time else omitted
- Added the profile image by default unless specified

### CODE REVIEW

**Review by andrew032011 — APPROVED** [review]

File: new-dti-website/src/app/course/page.tsx [hunk 1]
```diff
@@ -115,19 +115,19 @@ export default function Courses() {
...
                   MODERN INDUSTRY-LEADING TECHNOLOGY
                 </div>
 
-                <div className="font-black text-4xl tracking-wider mt-4 md:text-[45px]">
+                <div className="font-bold text-4xl mt-4 md:text-[45px]">
                   Trends in Web Development
                 </div>
 
-                <div className="text-md md:text-2xl mt-8">
+                <div className="text-md md:text-xl mt-8">
```
Comment (andrew032011) [inline new-dti-website/src/app/course/page.tsx:126]: @JasonMun7 I believe we're using `md:text-lg` for the body text on every other page. I tried to standardize it but looks like I missed it here!

Also side note but I don't think `text-md` refers to a valid CSS class :P, maybe remove it?

Comment (andrew032011) [inline new-dti-website/src/app/course/page.tsx:126]:
```suggestion
                <div className="md:text-lg mt-8">
```

Comment (JasonMun7) [reply new-dti-website/src/app/course/page.tsx:126]: Yes of course! Also you're right about `text-md` not being valid haha should be `text-base`! I also changed it for the other body text within the page as well to be md:text-lg! Thanks Andrew!

File: new-dti-website/src/app/course/page.tsx [hunk 2]
```diff
@@ -141,7 +141,7 @@ export default function Courses() {
                   <Link
                     key="Trends Website"
                     href={config.trendsWebsiteLink}
-                    className="primary-button"
+                    className="secondary-button secondary-button--red"
```
[the remark this answers is not in the document]
Comment (JasonMun7) [reply new-dti-website/src/app/course/page.tsx:144]: Sounds Good! Thanks @Bookie0!

**Pull request merged**